indico/modules/events/management/controllers/settings.py

```diff
@@ -139,6 +142,19 @@ class RHEditEventClassification(RHEditEventDataBase):
     form_class = EventClassificationForm
     section_name = 'classification'
 
+    def _process(self):
+        allowed_keywords = misc_settings.get('allowed_keywords')
+        if allowed_keywords:
```

Comment:
```suggestion
        if allowed_keywords := misc_settings.get('allowed_keywords'):
```

Reply:
OMG didn't know about this operator! ✨ 

I'll use it heavily from now on, thanks!

Reply:
another super common way to use sets for this type of code is this:

```py
if invalid := keywords - allowed_keywords:
    raise ValidationError(f'Invalid keywords: {", ".join(invalid)}')
```

(overkill here, and it might expose you to XSS since I think the ValidationError message is considered HTML, so let's not do that)

Reply:
Love it ❤️